src/components/Problems/Problems.tsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
+import { useRouter } from 'next/router';
+import { nullable } from '@taskany/bricks';
 
+import { parseNumber } from '../../utils/paramParsers';
 import { ProblemList } from '../ProblemList/ProblemList';
 import { LayoutMain } from '../LayoutMain/LayoutMain';
 import { ProblemFilterBar } from '../ProblemFilterBar/ProblemFilterBar';
 
 import { tr } from './Problems.i18n';
 
-export const Problems = () => (
-    <LayoutMain pageTitle={tr('Problems')} filterBar={<ProblemFilterBar title={tr('Problems')} />}>
-        <ProblemList />
-    </LayoutMain>
-);
+export const Problems = () => {
+    const router = useRouter();
+
+    const interviewId = parseNumber(router.query.interviewId);
```

Comment:
This page doesn't have interviewId and sectionId. Let's rename component to ProblemsPage to make it clear